examples/tutorials/Part 12 bis - Encrypted Training on MNIST.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<p>Is there an existing diagram explaining the third paragraph? I feel that I might have seen it in a tutorial... anyway, a diagram would be nice :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I have the figure you need, I'll add it right now :)</p>